HermesProxy/World/Server/Packets/AccountDataPackets.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using HermesProxy.Enums;
+using HermesProxy.World.Enums;
+
+namespace HermesProxy.World.Server.Packets
+{
+    public class AccountCharacterListEntry
+    {
+        public void Write(WorldPacket packet)
+        {
+            packet.WritePackedGuid128(AccountId);
+            packet.WritePackedGuid128(CharacterGuid);
+            packet.WriteUInt32(RealmVirtualAddress);
+            packet.WriteByteEnum(Race);
+            packet.WriteByteEnum(Class);
+            packet.WriteByteEnum(Sex);
+            packet.WriteUInt8(Level);
+
+            if (ModernVersion.AddedInVersion(ClientVersionBuild.V9_0_5_37503))
```

Comment:
Why are you checking retail build numbers? Those have no relation to classic.

Reply:
What build should I use instead? `V1_13_7_37279`?

Reply:
So just remove the version check.